metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql

```diff
@@ -276,4 +276,70 @@ JOIN
 WHERE
   SE.SCHEDULED_QUERY_ID=SQ.SCHEDULED_QUERY_ID;
 
+CREATE OR REPLACE VIEW `COMPACTIONS`
```

Comment:
Please add HIVE ticket id as comment, see line 132.

Reply:
Thanks for identifying this. Updated the PR.